src/screens/PasswordRecovery/componentes/GoBackToSignIn/styles.js

```diff
@@ -0,0 +1,14 @@
+import { StyleSheet } from 'react-native'
+import { verticalScale, scale } from 'react-native-size-matters/extend'
+
+const styles = StyleSheet.create({
+  text: {
+    textAlign: 'center',
+    fontFamily: 'Open Sans',
+    fontSize: scale(14),
+    lineHeight: verticalScale(14),
+    color: '#FE9458'
```

Comment:
I do not think this works (I can't recall if Open Sans is a font that comes in Android, in case it isn't, it rollbacks to the default font which is Roboto). Regardless, I would stick to "Red hat Display", which is applied automatically to the Body component. I know some components in the design have Open Sans, but it breaks consistency with the original designs made by Agos. Also change `'#FE9458'` to `theme.colors.secondary` and remove `fontSize`

@nahudalla I would like your opinion on this matter too.